app/controllers/supervisor/courses_controller.rb

```diff
@@ -24,7 +24,11 @@ def create
 
   def edit; end
 
-  def show; end
+  def show
+    @course_subjects = CourseSubject.where(course_id: @course.id)
+    @added_subject_ids = @course_subjects.pluck(:subject_id)
+    @subjects = Subject.where.not(id: @added_subject_ids)
```

Comment:
`.where.not(id: @added_subject_ids)` scope hì

ở trên line 28 nữa

Reply:
fixed